command/aarelayer/service/aa_types_test.go

```diff
@@ -0,0 +1,47 @@
+package service
+
+import (
+	"encoding/hex"
+	"math/big"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo/wallet"
+)
+
+func TestE2E_MakeSignature_And_GetAddressFromSignature(t *testing.T) {
```

Comment:
Not sure why it is called TestE2E, since it is a regular unit test
```suggestion
func Test_MakeSignature_And_GetAddressFromSignature(t *testing.T) {
```

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1284/commits/3d92de70afa94706f3002144958fe9b5783ba13c

Reply:
Sorry for being boring, but now it makes sense to adapt the name of the test to be `Test_Sign_And_RecoverSender` since function names are changed.